js/modules/loki_app_dot_net_api.js

```diff
@@ -1705,18 +1732,31 @@ class LokiPublicChannelAPI {
     // filter out invalid messages
     pendingMessages = pendingMessages.filter(messageData => !!messageData);
     // separate messages coming from primary and secondary devices
-    const [primaryMessages, slaveMessages] = _.partition(
+    let [primaryMessages, slaveMessages] = _.partition(
       pendingMessages,
       message => !(message.source in slavePrimaryMap)
     );
-    // process primary devices' message directly
-    primaryMessages.forEach(message =>
-      this.chatAPI.emit('publicMessage', {
-        message,
-      })
-    );
-
-    pendingMessages = []; // allow memory to be freed
+    // get minimum ID for primaryMessages and slaveMessages
+    const firstPrimaryId = Math.min(...primaryMessages.map(msg => msg.serverId));
```

Comment:
Lodash has a `_.min` function which takes in an array. We could use it here to make it less confusing. Thoughts?